core/textinput/src/Getline.cxx

```diff
@@ -85,6 +85,23 @@ namespace {
    };
    const size_t ROOTTabCompletion::fgLineBufSize = 16*1024;
 
+   class TClingTabCompletion: public TabCompletion {
+   public:
+      TClingTabCompletion() {}
+      virtual ~TClingTabCompletion() {}
+
+      TClingTabCompletion(const TClingTabCompletion&) = delete;
+      TClingTabCompletion& operator=(const TClingTabCompletion&) = delete;
+
+      // Returns false on error
+      bool Complete(Text& line /*in+out*/, size_t& cursor /*in+out*/,
+                    EditorRange& r /*out*/,
+                    std::vector<std::string>& displayCompletions /*out*/) {
```

Comment:
Sure, but I thought the name comes from the Display class, because it is the one processing the completions to print them. I can change it if course :)